cayleypy/graphs_lib.py

```diff
@@ -44,6 +44,21 @@
     "M_Rvi": [4, 5, 3, 11, 9, 10, 7, 8, 6, 0, 1, 2, 20, 18, 19, 14, 12, 13, 17, 15, 16, 23, 21, 22],
 }
 
+# fmt: off
+# pylint: disable=line-too-long
+PYRAMINX_MOVES = {
```

Comment:
IMHO no need to disable formatting here, it will look just fine. Yes, more lines - but easier to read, especially when looking at diff on GiHub. But you can keep this.